browser/brave_shields/ad_block_only_mode_browsertest.cc

```diff
@@ -0,0 +1,230 @@
+
+    EXPECT_TRUE(profile_prefs()->GetBoolean(de_amp::kDeAmpPrefEnabled));
+    EXPECT_FALSE(
+        profile_prefs()->IsManagedPreference(de_amp::kDeAmpPrefEnabled));
+
+    EXPECT_TRUE(profile_prefs()->GetBoolean(debounce::prefs::kDebounceEnabled));
+    EXPECT_FALSE(profile_prefs()->IsManagedPreference(
+        debounce::prefs::kDebounceEnabled));
+
+    EXPECT_TRUE(profile_prefs()->GetBoolean(
+        query_filter::kTrackingQueryParametersFilteringEnabled));
+    EXPECT_FALSE(profile_prefs()->IsManagedPreference(
+        query_filter::kTrackingQueryParametersFilteringEnabled));
+
+    EXPECT_TRUE(profile_prefs()->GetBoolean(
+        global_privacy_control::kGlobalPrivacyControlEnabled));
+    EXPECT_FALSE(profile_prefs()->IsManagedPreference(
+        global_privacy_control::kGlobalPrivacyControlEnabled));
+  }
+
+  void VerifyAdblockOnlyModeParameters() {
+    EXPECT_EQ(GetNoScriptControlType(host_content_settings_map(), GURL()),
+              ControlType::ALLOW);
+
+    EXPECT_EQ(GetCookieControlType(host_content_settings_map(),
+                                   cookie_settings().get(), GURL()),
+              ControlType::ALLOW);
+
+    EXPECT_EQ(GetFingerprintingControlType(host_content_settings_map(), GURL()),
+              ControlType::ALLOW);
+
+    EXPECT_FALSE(profile_prefs()->GetBoolean(prefs::kReduceLanguageEnabled));
+    EXPECT_TRUE(
+        profile_prefs()->IsManagedPreference(prefs::kReduceLanguageEnabled));
+
+    EXPECT_FALSE(profile_prefs()->GetBoolean(de_amp::kDeAmpPrefEnabled));
+    EXPECT_TRUE(
+        profile_prefs()->IsManagedPreference(de_amp::kDeAmpPrefEnabled));
+
+    EXPECT_FALSE(
+        profile_prefs()->GetBoolean(debounce::prefs::kDebounceEnabled));
+    EXPECT_TRUE(profile_prefs()->IsManagedPreference(
+        debounce::prefs::kDebounceEnabled));
+
+    EXPECT_FALSE(profile_prefs()->GetBoolean(
+        query_filter::kTrackingQueryParametersFilteringEnabled));
+    EXPECT_TRUE(profile_prefs()->IsManagedPreference(
+        query_filter::kTrackingQueryParametersFilteringEnabled));
+
+    EXPECT_FALSE(profile_prefs()->GetBoolean(
+        global_privacy_control::kGlobalPrivacyControlEnabled));
+    EXPECT_TRUE(profile_prefs()->IsManagedPreference(
+        global_privacy_control::kGlobalPrivacyControlEnabled));
+  }
+
+ private:
+  PrefService* profile_prefs() { return GetProfile()->GetPrefs(); }
+
+  HostContentSettingsMap* host_content_settings_map() {
+    return HostContentSettingsMapFactory::GetForProfile(GetProfile());
+  }
+
+  scoped_refptr<content_settings::CookieSettings> cookie_settings() {
+    return CookieSettingsFactory::GetForProfile(GetProfile());
+  }
+};
+
+class AdblockOnlyModeBrowserTest : public AdblockOnlyModeBrowserTestBase {
+ public:
+  AdblockOnlyModeBrowserTest() {
+    scoped_feature_list_.InitAndEnableFeature(features::kAdblockOnlyMode);
+  }
+
+  ~AdblockOnlyModeBrowserTest() override = default;
+
+ private:
+  base::test::ScopedFeatureList scoped_feature_list_;
```

Comment:
nit: you can pass `features::kAdblockOnlyMode` right here `{features::kAdblockOnlyMode`}`

Reply:
Thanks for comment! Fixed